function_computer_grid.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed May 13 13:16:43 2020
+
+@author: carolinstreitberger
+"""
+import numpy as np
+import random
+
```

Comment:
Your documentation is among the clearest I've ever seen. It's honestly great and easy to follow!